themes/bootstrap3/js/search.js

```diff
@@ -293,7 +293,7 @@ VuFind.register('search', function search() {
     loadingOverlay.classList = 'loading-overlay';
     loadingOverlay.setAttribute('aria-live', 'polite');
     loadingOverlay.setAttribute('role', 'status');
-    loadingOverlay.innerHTML = VuFind.loading();
+    VuFind.setInnerHtml(loadingOverlay, VuFind.loading());
```

Comment:
Would it make more sense to have `VuFind.loading` and `VuFind.spinner` return elements instead of HTML and use `append` or `replaceChildren` here?

Reply:
Maybe it would be even better to have `loadingElements` and `spinnerElements` methods instead of changing the existing `loading` and `spinner` for backward-compatibility. We could always mark those as deprecated and remove them later if they're not widely used.

Reply:
Sure. ill adjust a lttilebit more, might take a while but in close future :) will request re reviews when adjusted